pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,253 @@
+               data-show-button-text="true"
+               data-pagination="true"
+               data-pagination-v-align="top"
+               data-pagination-detail-h-align="right"
+               data-page-list="[10,20,50,100,200,500,unlimited]"
+               data-page-size="50"
+               data-smart-display="false"
+               data-show-extended-pagination="true"
+               data-toolbar=".fixed-table-pagination"
+               data-sticky-header="true">
+          <thead>
+            <tr>
+              <th data-field="qid"
+                  data-sortable="true"
+                  data-class="align-middle sticky-column"
+                  data-formatter="qidFormatter"
+                  data-filter-control="input"
+                  data-filter-custom-search="genericFilterSearch"
+                  data-switchable="true">QID</th>
+              <th data-field="title"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-filter-control="input"
+                  data-switchable="true">Title</th>
+              <th data-field="topic"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="topicFormatter"
+                  data-sorter="topicSorter"
+                  data-filter-control="select"
+                  data-filter-control-placeholder="(All Topics)"
+                  data-filter-data="func:topicList"
+                  data-filter-custom-search="badgeFilterSearch"
+                  data-switchable="true">Topic</th>
+              <th data-field="tags"
+                  data-sortable="false"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="tagsFormatter"
+                  data-filter-control="select"
+                  data-filter-control-placeholder="(All Tags)"
+                  data-filter-data="func:tagsList"
+                  data-filter-custom-search="badgeFilterSearch"
+                  data-switchable="true">Tags</th>
+              <th data-field="display_type"
+                  data-sortable="true"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="versionFormatter"
+                  data-filter-control="select"
+                  data-filter-control-placeholder="(All Versions)"
+                  data-filter-data="func:versionList"
+                  data-filter-custom-search="badgeFilterSearch"
+                  data-visible="<%= has_legacy_questions %>"
+                  data-switchable="true">Version</th>
+              <% locals.course_instances.forEach(course_instance => { %>
+              <th data-field="assessments_<%= course_instance.id %>"
+                  data-class="align-middle text-nowrap"
+                  data-formatter="assessments<%= course_instance.id %>Formatter"
+                  data-filter-control="select"
```

Comment:
Any chance of a natural sort here?

<img width="244" alt="image" src="https://user-images.githubusercontent.com/875833/129056078-acbe3164-b1e7-43a3-bad9-eabad0fab434.png">

Reply:
I tried playing around with this, but I believe there's a bug in the extension that is ignoring the [`data-filter-order-by`](https://bootstrap-table.com/docs/extensions/filter-control/#filterorderby) option and always sorting in ascending order of text representation. Sorry, if there is a solution, it would be hacky and hard.

Reply:
No worries, we can just forget about this. Thanks for trying!